### ISSUE TITLE
[Feature request] Show tpe of custom field

### ISSUE THREAD

**Issue by fdw** [doy/rbw#67]
In Bitwarden, a custom field on an entry can have one of three types: `text`, `hidden` and `boolean`. Can `rbw get --full` maybe also return this type?

**Comment by fdw** [comment]
So, I have been playing with this, but II'm very new to Rust, and so I need some help to implement this. Would that still be okay for you?

Currently, I'm stuck because the synced data does not contain the type of the field, even though it should be there. I also couldn't find a documentation about Bitwarden's sync API. Do you have any pointers?

**Comment by fdw** [comment]
I did make some progress 🎉

The next question would be: How to display the field type? Do you have any good idea?

**Comment by fdw** [comment]
Hi @doy, I just wanted to bump this 🙈 Do you have any preference how to display the field type?

**Comment by fdw** [comment]
Is this still something you're interested in? Can we also get the type into `--raw`?

**Comment by therealchfkch** [comment]
Hello @fdw and @doy, i am building a bitwarden linux mobile client and using rbw as a backend. I also would like to have the type of the custom field since i need to hide it in case of "hidden". Is there anything i can help out with?
I have analyzed a good portion of rbw already, so i might be able to help with the implementation. Otherwise i would start from scratch.
I have seen there is already https://github.com/doy/rbw/pull/89 is this still up to date?

Link to the app: https://codeberg.org/Chfkch/bitritter

**Comment by fdw** [comment]
No, #89 is not up to date; it's a bit more than two years old. If I find time, I can maybe update it, unless you want to.

**Comment by therealchfkch** [comment]
@fdw i do not have much time until July/August, so if you want to finish it before, go ahead. Otherwise i might pick you PR up and see if it can be ported to the latest version.
